docs/contributing/how-to-release.md

```diff
@@ -46,23 +46,22 @@ and then run it using the new DLLs.
 
 ## Pre-Release Steps
 
-* Update `docs/readme.txt`. Please review its contents in addition to just
-  changing the version number.
-* Update `docs/release.md` (the release sha1sums should be set to zeroes).
-* Put a date on the release line in `docs/changes.txt`.
-* Update the date in the manual (`docs/doxygen/mainpages/manual.h`).
-* Update the release announcement post in `docs/publicity/announce.txt`.
+1. Perform the following steps. You can run `build/tools/pre-release.sh` to do
+   the straightforward changes like updating the date and version number
+   automatically, but please also review and update the contents of the README
```

Comment:
IIUC `pre-release.sh` will take care of the points below, in which case shouldn't "review ~~and update~~ the contents" be enough? Otherwise what else should be updated?
`README` is probably the same as `docs/readme.txt`, right? And `announcement text` is probably `docs/publicity/announce.txt`, but stopping at `docs/release.md` was so close.
Actually shouldn't the advice be more like "review the changes to affected files before committing", even if that should already be redundant for any programmer?

Reply:
There are parts of these files that need to be manually updated for each release, e.g. the changes part. Maybe we can just use the release notes as the announcement text, but it looks a bit too long for this to me... OTOH it's true that currently there is significant overlap between them.